package.json

```diff
@@ -21,36 +21,36 @@
     "vue": "^2.0.0"
   },
```

Comment:
build is failing

Reply:
Needed a newer version of NodeJS.
I just pushed a commit to update Travis to Node 14 for this repo, that should sort the build.